packages/serializer-md/src/deserializer/utils/deserializeMd.spec.tsx

```diff
@@ -294,7 +294,9 @@ describe('deserializeMd', () => {
       <fragment>
         <hp>
           <htext>Line 1</htext>
-          <htext>{'\n'}</htext>
+          <hp>
+            <htext></htext>
+          </hp>
```

Comment:
Nesting paragraphs isn't valid HTML markup. I think `\n` is the only natural choice for deserializing a `<br>` element.

Reply:
Yes, but how you will detect `\n` when serialising it back to `<br>`?

Reply:
You could convert all `\n` characters occuring in text nodes to `<br>`, or just keep them as `\n`s. That would be the most natural choice for the majority of Plate's users; if you need something different in your app, it might be best to do that in your own source code.

Reply:
Ok, will update it